data/hai/midas.txt

```diff
@@ -0,0 +1,940 @@
+		payment 4000000
+		dialog
+			`As before, a voice addresses you once you are alone in your quarters. "We have transferred 4 million credits to your account, and made sure the Chief will not bother you.`
+			`	"Do be aware, however, that a food situation has arrisen, not because of our small operation, but because of the bomb's presence in the first place.`
+			`	"The Unfettered will probably be busy for some time... As for the governor, if it can ease your conscience, we only captured her."`
+
+
+
+# secret mission
+# happens if you accept a duel in "<uhai> 7: Stolen Cargo" (offered if your fleet is smaller than 2 heavy warships)
+mission "<uhai> 7: Duel"
+	landing
+	source "New Tortuga"
+	name "Duel with the twins."
+	description "Beat the two twins today, or lose 10 million credits!"
+	to offer
+		has "midas: accepted duel"
+		has "<uhai> 7: Cavalry Arrives: done"
+		"unfettered: cleaned up cerberus" > 1
+	on offer
+		dialog `The two mysterious ships hail you, "We are ready for a fight in orbit whenever you are. If you want, you can still call this off... for a quarter of the agreed price."`
+	on decline
+		fine 2500000
+	npc disable save
+		government "Hai (Unfettered Challenger)"
+		to despawn
+			has "despawn the duel ships"
+		personality waiting heroic unconstrained staying disables
+		ship "Sea Dragon" "Thorn"
+		ship "Sea Dragon" "Talon"
+		on kill
+			dialog "You killed one of the ships. You will need to restart the mission in order to progress further with the Shadow of Mintaka. They may want to have their vengeance on you... The Unfettered may not offer you any missions anymore, for disrespecting the sacred rules of the duel."
+			set "midas: thorn talon death"
+			set "midas: thorn talon murdered"
+			"midas: wronged" ++
+			event "midas: thorn talon vengeance" 180
+			log `Destroyed mysterious ships that serve the 'Sadow of Mintaka' in a duel. What's the worst that could happen? No idea if the Unfettered will forgive this act.`
+		conversation
+			`Seeing the fight is over, they hail you, "Indeed you know how to fight, <last>," the first one compliments you, "That was just luck!" the other one argues. "Whatever the case, you have gained the right to plunder our ships. But do not dare even think about trying to capture. We will be leaving once you land. Oh, and here are your 10 million credits back."`
+			action
+				set "midas: won the duel"
+				log `Won a duel against two mysterious ships that serve the 'Shadow of Mintaka', whatever that means. Plundering of their ships was so nicely offered!`
+				payment 10000000
+	on enter
+		set "despawn the duel ships"
+	on abort
+		dialog `You decide to abort the duel. They hail you, "We have no idea why you decided to cancel the duel, but our prize of 10 million credits is still due."`
+		log `Lost a duel against two mysterious ships that serve the 'Shadow of Mintaka', whatever that means. Had to pay up 10 million credits.`
```

Comment:
It would probably be a good idea to add a disabled check to this mission, also.

Reply:
True, thats a very recent addition that I was thinking of adding everywhere but I do need to change the duel lore a bit first